index.html

```diff
@@ -183,7 +184,8 @@ <h2>Please choose a manuscript to load:</h2>
                 $("#serverLoadModal-primary").on('click', function()
                 {
                     var pageName = $("#selecthosted-file").find(':selected').text();
-                    $.get("http://" + currentSite + "/mei/" + siglum_slug + "/" + pageName, "", function(data)
+                    //$.get("https://" + currentSite + "/mei/" + siglum_slug + "/" + pageName, "", function(data)
+                    $.get("https://" + currentSite + "/resources/mei/csg-390/" + pageName, "", function(data)
```

Comment:
Why are we hardcoding 390 instead of making the app generic?

Reply:
Whenever we add other manuscripts, we need to decide which of the following structures is going to be

```
<root>/resources/mei/<manuscript>/<page>.mei
```
or
```
<root>/resources/mei/<manuscript>-<page>.mei
```
The naming convention you added to the repo follows the first example, the code works better with the second one (because matches the names you can fetch from IIIF), so I changed to the second, but I left the root folder (csg-390). Basically, for now, I am not deciding what it is going to be, I think looking at other manuscripts/mei files fetched from different places will help to know what is the usual naming convention. My priority, for now, has been to make the demo with 390 work with the features we expect, after that, I could come back to this issue. What do you think?

Reply:
Let's work, then, with the naming structure that matches this manuscript and open an issue to study how we can generalize it for different manuscripts.

Reply:
That works, I will make a few changes to reflect that and push it with the next pull request.